core/src/test/scala/pureconfig/PureconfSuite.scala

```diff
@@ -51,6 +51,11 @@ class PureconfSuite extends FlatSpec with Matchers with OptionValues with Either
     }
   }
 
+  "ConfigConvert" should s"be able to convert from and to None" in {
```

Comment:
Fixed, thanks for pointing out when I add the `s` without reason. I think I'm doing it a lot and it's completely involuntary.